snap/snapcraft.yaml

```diff
@@ -6,15 +6,7 @@ description: |
   and traces from various sources to multiple destinations.
 contact: jose.masson@canonical.com
 base: core24
-platforms:
-  amd64:
-    build-on: [amd64]
-    build-for: [amd64]
-  arm64:
-    build-on: [arm64]
-    build-for: [arm64]
```

Comment:
This can probably be simplified to 
```
platforms:
  amd64:
  arm64:
```

but probably shouldn't be removed entirely?